src/mock/test/BalanceTest.cpp

```diff
@@ -1,182 +0,0 @@
-/* Copyright (c) 2020 vesoft inc. All rights reserved.
```

Comment:
This need tests too.

Reply:
The test environment is now not clustered, The balancer testing is meaningless.